vendor/data/codelists/IATI/2_03/activity/recipient_country.yml

```diff
@@ -0,0 +1,765 @@
+---
```

Comment:
Is there some process that took this to create the values in i18n?

Reply:
Me downloading the csv and converting it to yaml?

Reply:
I mean to get from here to the locale file.

Reply:
Sounds like Meyric did this by hand. FYI we made a script for this that we can use locally https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/lib/tasks/yaml_to_locale.rake